docs/reference/ml/anomaly-detection/ml-configuring-aggregations.asciidoc

```diff
@@ -134,25 +138,134 @@ PUT _ml/datafeeds/datafeed-farequote
 ----------------------------------
 // TEST[skip:setup:farequote_job]
 
-<1> The aggregations have names that match the fields that they operate on. The 
+<1> The aggregations have names that match the fields that they operate on. The
 `max` aggregation is named `time` and its field also needs to be `time`.
-<2> The `term` aggregation is named `airline` and its field is also named 
+<2> The `term` aggregation is named `airline` and its field is also named
 `airline`.
-<3> The `avg` aggregation is named `responsetime` and its field is also named 
+<3> The `avg` aggregation is named `responsetime` and its field is also named
 `responsetime`.
 
-Your {dfeed} can contain multiple aggregations, but only the ones with names 
-that match values in the job configuration are fed to the job.
+TIP: If you are utilizing a `term` aggregation to gather influencer or partition
+detector field information, consider using a `composite` aggregation. It performs
```

Comment:
> partition detector field information...

Is this the partition_field_name value from the job config?  I've never seen it called a "partition detector field", so unless this is something else, I'd recommend sticking with just "partition field".

Reply:
yeah, I think my brain just wanted to make word salad :)